examples/example-server-lib/tiling_window_manager.cpp

```diff
@@ -21,7 +21,7 @@
 #include <miral/application_info.h>
 #include <miral/window_info.h>
 #include <miral/window_manager_tools.h>
-#include <miral/output.h>
+#include <miral/zone.h>
```

Comment:
The changes to TilingWindowManagerPolicy are simplistic.

The policy takes the bounding rectangle of "displays" and lays out the "tiles" within that. That means it will tile over any gaps between "application zones" which is unlikely to be the intent.

In part this is pre-existing (as if the outputs are not arranged in a rectangle then areas outside them will be used). So this isn't a blocker, just a note.

Reply:
At least this will still make it work with exclusive zones if there is only one monitor attached. It may not be perfect, but I'd call that preexisting and this an improvement.